pkg/cmd/event/event.go

```diff
@@ -0,0 +1,64 @@
+package event
+
+import (
+	"context"
+	"fmt"
+	"github.com/jedib0t/go-pretty/table"
+	"github.com/puppetlabs/nebula-cli/pkg/client"
+	"github.com/puppetlabs/nebula-cli/pkg/config/runtimefactory"
+	"github.com/spf13/cobra"
+)
+
+func NewCommand(rt runtimefactory.RuntimeFactory) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                   "event",
```

Comment:
Might want to call this `event-sources` instead, as I could definitely see us wanting to list individual events somewhere down the line too.